pints/_mcmc/_adaptive_covariance_am.py

```diff
@@ -0,0 +1,68 @@
+#
+# Adaptive covariance MCMC method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class AdaptiveCovarianceAMMCMC(pints.AdaptiveCovarianceMCMC):
+    """
+    Adaptive Metropolis MCMC, as described by Algorithm 2 in [1],
+    (with gamma = self._adaptations ** -eta which isn't specified
```

Comment:
You can just call this variable `adaptations` in the comments. The fact that it's a hidden variable (`_`) stored in the object (`self.`) isn't relevant to the user reading this